requirements-test.txt

```diff
@@ -0,0 +1,3 @@
+pytest==3.0.2
+pytest-xdist==1.15.0
+flaky
```

Comment:
Given our issues with dependencies shifting silently in the past, any reason not to pin the version of flaky?